uuid.go

```diff
@@ -46,7 +46,7 @@ type UUID [Size]byte
 const (
 	_  byte = iota
 	V1      // Version 1 (date-time and MAC address)
-	V2      // Version 2 (date-time and MAC address, DCE security version)
+	_       // Version 2 (date-time and MAC address, DCE security version) [removed]
```

Comment:
I would leave the constant there. There is no reason to stop users from being able to identify that this is a v2 UUID.

```suggestion
	V2      // Version 2 (date-time and MAC address, DCE security version) [not supported]
```

Reply:
What makes you believe removing this stops users from being able to identify a UUIDv2? I think it makes more sense for them to now define a constant in their project instead of rely on ours for it, since we no longer provide V2 support.

Reply:
To clarify, the type here is a `byte`. It should be trivial for someone to define their own constant and use it. It would also avoid people using our package just for the `V2` constant.

Reply:
This isn't that important, so it is fine to remove.